### DISCUSSION TITLE
New Feature: Custom Fonts

### DISCUSSION THREAD

**Discussion by steven-tey** [steven-tey/novel#67] [Announcements]
Just added custom fonts to Novel ✨

https://github.com/steven-tey/novel/assets/28986134/1c140bad-54a7-4057-99d7-a68d9a694d00

Starting with Sans Serif & Serif for now, but might add more good ones in the future!

What are some fonts that you'd like to see on Novel? 👀

**Comment by sebi75** [comment]
Cool! What are you using for recording?

**Reply by Amsalak** [reply]
Kayıt için derken

**Reply by steven-tey** [reply]
Used ScreenStudio for this one!
